### PR TITLE
SEO-1917-winforms-combobox-money-keywords.

### PR DESCRIPTION
**Pull request by Aishwarya-Ganesan**
Hi @ChristoIssac,

Please find the winforms combobox money keywords changes.

Regards,
@Aishwarya-Ganesan.

### CODE REVIEW

File: WindowsForms/ComboBox/GettingStarted.md [hunk 1]
```diff
@@ -152,3 +152,4 @@ sfComboBox1.ComboBoxMode = ComboBoxMode.MultiSelection
 
 Index of the selected item can be retrieved by using the [SelectedIndex](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.ListView.SfComboBox.html#Syncfusion_WinForms_ListView_SfComboBox_SelectedIndex) property. The value of the selected item can be retrieved by using the [SelectedValue](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.ListView.SfComboBox.html#Syncfusion_WinForms_ListView_SfComboBox_SelectedValue) property. It returns the value of the property bound to the `ValueMember` property. If the `ValueMember` is not initialized, it will return the value of the property bound to the `DisplayMember`. The selected item of the SfComboBox can be retrieved by using the [SelectedItem](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.ListView.SfComboBox.html#Syncfusion_WinForms_ListView_SfComboBox_SelectedItem) property.
 
+N> N> You can also explore our [WinForms ComboBox example](https://github.com/syncfusion/winforms-demos/tree/master/sfcombobox) that shows how to allows users to type a value, choose an item, or choose multiple items using a multiselect option.
```
Comment (Girija-Rajendran) [inline WindowsForms/ComboBox/GettingStarted.md:155]: "Note" syntax error

File: WindowsForms/ComboBox/Overview.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Windows Forms ComboBox (SfComboBox) Overview
 
-Combo box is a text box component that allows user to type a value or choose an option from a list of predefined options. It has several out of the box features such as data binding, filtering, UI customization, accessibility, and more.
+[Combo box](https://www.syncfusion.com/winforms-ui-controls/combobox) is a text box component that allows user to type a value or choose an option from a list of predefined options. It has several out of the box features such as data binding, filtering, UI customization, accessibility, and more.
```
Comment (Girija-Rajendran) [inline WindowsForms/ComboBox/Overview.md:12]: Add the platform name before the control name

File: WindowsForms/ComboBox/Selection.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ End Sub
 
 ## Multi-selection
 
-The SfComboBox allows you to select multiple values from the drop-down list. The multi-select combo box mode can be enabled by setting the [ComboBoxMode](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.ListView.SfComboBox.html#Syncfusion_WinForms_ListView_SfComboBox_ComboBoxMode) property as `MultiSelection`.
+The SfComboBox allows you to select multiple values from the drop-down list. The [multi-select combo box](https://www.syncfusion.com/winforms-ui-controls/combobox/multiselect-combobox) mode can be enabled by setting the [ComboBoxMode](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.ListView.SfComboBox.html#Syncfusion_WinForms_ListView_SfComboBox_ComboBoxMode) property as `MultiSelection`.
```
Comment (Girija-Rajendran) [inline WindowsForms/ComboBox/Selection.md:73]: Add the platform name before the control name

File: WindowsForms/Ribbon/Getting-Started.md [hunk 4]
```diff
@@ -1807,4 +1807,4 @@ N> ApplicationMenu can be accessed only when ribbon style is Office2007
 
 The simplified layout is designed to display the most commonly used Ribbon commands in a single line interface, allowing more screen space for compact content viewing, while other commands are placed inside the overflow menu. To know more about the simplified layout, refer [here](https://help.syncfusion.com/windowsforms/ribbon/simplifiedlayout).
 
-
+N> You can also explore our [WinForms Ribbon example](https://github.com/syncfusion/winforms-demos/tree/master/ribbon) that shows how to accommodates all the tools required for an application in a single, easy-to-navigate user interface similar to Microsoft Office.
```
Comment (Girija-Rajendran) [inline WindowsForms/Ribbon/Getting-Started.md:1810]: Could you include a content review task for this?

File: WindowsForms/Ribbon/Overview.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Windows Forms Ribbon (RibbonControlAdv) Overview
 
-The WinForms [RibbonControlAdv](https://help.syncfusion.com/cr/windowsforms/Syncfusion.Windows.Forms.Tools.RibbonControlAdv.html) (ribbon menu) accommodates all the tools required for an application in a single, easy-to-navigate user interface like Microsoft Office. Its dynamic resizing, keyboard accessibility, and complete customization options help to organize an application. It provides an application UI like Microsoft Office with a 2007 style that contains an application menu and 2016 style that contains a backstage view.
+The [WinForms Ribbon Control](https://help.syncfusion.com/windowsforms/ribbon/overview) (ribbon menu) accommodates all the tools required for an application in a single, easy-to-navigate user interface like Microsoft Office. Its dynamic resizing, keyboard accessibility, and complete customization options help to organize an application. It provides an application UI like Microsoft Office with a 2007 style that contains an application menu and 2016 style that contains a backstage view.
```
Comment (Girija-Rajendran) [inline WindowsForms/Ribbon/Overview.md:12]: Why was the cr link removed here?

File: WindowsForms/ComboBox/GettingStarted.md (hunk 1 above)
Comment (Aishwarya-Ganesan) [reply WindowsForms/ComboBox/GettingStarted.md:155]: changed

File: WindowsForms/ComboBox/Selection.md (hunk 3 above)
Comment (Aishwarya-Ganesan) [reply WindowsForms/ComboBox/Selection.md:73]: Added

File: WindowsForms/ComboBox/Overview.md (hunk 2 above)
Comment (Aishwarya-Ganesan) [reply WindowsForms/ComboBox/Overview.md:12]: Added

File: WindowsForms/Ribbon/Getting-Started.md (hunk 4 above)
Comment (Aishwarya-Ganesan) [reply WindowsForms/Ribbon/Getting-Started.md:1810]: I have taken from demo only.

File: WindowsForms/Ribbon/Overview.md (hunk 5 above)
Comment (Aishwarya-Ganesan) [reply WindowsForms/Ribbon/Overview.md:12]: Not opening properly so only removed

**Review by Girija-Rajendran — APPROVED** [review]

**Review by ChristoIssac — APPROVED** [review]

**Pull request merged**